code/modules/clothing/under/ties.dm

```diff
@@ -367,3 +371,35 @@
 	icon_state = "talisman"
 	item_color = "talisman"
 	armor = list(melee = 5, bullet = 5, laser = 5, energy = 5, bomb = 20, bio = 20, rad = 5, fire = 0, acid = 25)
+
+//////////////
+//OBJECTION!//
+//////////////
+
+/obj/item/clothing/tie/lawyers_badge
+	name = "attorney's badge"
+	desc = "Fills you with the conviction of JUSTICE. Lawyers tend to want to show it to everyone they meet."
+	icon_state = "lawyerbadge"
+	item_color = "lawyerbadge"
+
+/obj/item/clothing/tie/lawyers_badge/attach(obj/item/clothing/under/U, user)
+	if(!..())
+		return 0
+	if(isliving(U.loc))
+		on_uniform_equip(U)
+
+/obj/item/clothing/tie/lawyers_badge/detach(obj/item/clothing/under/U, user)
+	..()
+	if(isliving(U.loc))
+		on_uniform_dropped(U)
+
+/obj/item/clothing/tie/lawyers_badge/on_uniform_equip(obj/item/clothing/under/U)
+	if(!isliving(U.loc))
+		return
+	var/mob/living/L = U.loc
+	L.bubble_icon = "lawyer"
+
+/obj/item/clothing/tie/lawyers_badge/on_uniform_dropped(obj/item/clothing/under/U)
+	if(!isliving(U.loc))
+		return
+	var/mob/living/L = U.loc

```

Comment:
Good catch. Somehow the terminal line was lost in the merge resolution. Been fixed.